src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -75,6 +75,11 @@ export default {
     mounted() {
         this.updateView();
     },
+    beforeDestroy() {
+        if (this.component) {
```

Comment:
i see another method, `updateView` destroying `this.component`. It also does a little more cleanup. Do we need to do the same here? If so, can we make a `destroyComponent` method?